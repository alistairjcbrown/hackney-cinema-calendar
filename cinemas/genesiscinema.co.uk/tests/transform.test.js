const fs = require("node:fs");
const path = require("node:path");
const transform = require("../transform");

const readMockData = (filename) =>
  fs.readFileSync(
    path.resolve(__dirname, `./mock-data/${filename}.txt`),
    "utf8",
  );

const moviesWithLocalScreenings = readMockData("movie-with-local-screenings");

jest.useFakeTimers().setSystemTime(new Date("2024-10-14"));

// Disable caching for test
jest.mock("../../../common/cache", () => ({
  dailyCache: (key, retrieve) => retrieve(),
}));

// Mock retrieve calls for additional data
jest.mock("../retrieve", () => (url) => {
  const id = url.split("event/")[1].split("/")[0];
  return readMockData(`show-${id}`);
});

describe("The Genesis Cinema", () => {
  describe("when no movies provided", () => {
    it("returns an empty list of events", async () => {
      const value = "";
      expect(await transform(value)).toEqual([]);
    });
  });

  describe("when movies with local screenings provided", () => {
    it("returns a list of events", async () => {
      const value = moviesWithLocalScreenings;
      expect(await transform(value)).toMatchSnapshot();
    });
  });
});
