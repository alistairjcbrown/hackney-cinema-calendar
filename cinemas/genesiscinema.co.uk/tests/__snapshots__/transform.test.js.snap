// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Genesis Cinema when movies with local screenings provided returns a list of events 1`] = `
[
  {
    "overview": {
      "actors": [
        "Sebastian Stan",
        "Renate Reinsve",
        "Adam Pearson",
      ],
      "age-restriction": "15",
      "categories": [
        "Comedy",
        "Horror",
        "Thriller",
      ],
      "directors": [
        "Aaron Schimberg",
      ],
      "duration": 6720000,
      "trailer": "https://www.youtube.com/watch?v=3gIgpbZKltg",
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=567",
        "notes": "",
        "time": 1729022400000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=581",
        "notes": "",
        "time": 1729108800000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=592",
        "notes": "",
        "time": 1729184400000,
      },
    ],
    "title": "A Different Man",
    "url": "https://www.genesiscinema.co.uk/event/90175",
  },
  {
    "overview": {
      "actors": [
        "Mia Pérez Ullod",
        "Jaume Solà",
        "Javier Cassi Gimeno",
      ],
      "age-restriction": "PG",
      "categories": [
        "Comedy",
        "Adventure",
        "Family",
      ],
      "directors": [
        "Juan Jesús García Galocha",
        "Pedro Solís García",
      ],
      "duration": 5580000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=554",
        "notes": "",
        "time": 1729003500000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=568",
        "notes": "",
        "time": 1729089900000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=582",
        "notes": "",
        "time": 1729177500000,
      },
    ],
    "title": "Buffalo Kids",
    "url": "https://www.genesiscinema.co.uk/event/91614",
  },
  {
    "overview": {
      "actors": [
        "Joaquin Phoenix",
        "Zazie Beetz",
        "Lady Gaga",
      ],
      "age-restriction": "15",
      "categories": [
        "Crime",
        "Drama",
        "Thriller",
      ],
      "directors": [
        "Todd Phillips",
      ],
      "duration": 8280000,
      "trailer": "https://www.youtube.com/watch?v=xy8aJw1vYHo",
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=556",
        "notes": "",
        "time": 1729000800000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=502",
        "notes": "subtitles",
        "time": 1729011000000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=555",
        "notes": "",
        "time": 1729021800000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=571",
        "notes": "",
        "time": 1729087200000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=569",
        "notes": "",
        "time": 1729097400000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=570",
        "notes": "",
        "time": 1729108200000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=585",
        "notes": "",
        "time": 1729175400000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=590",
        "notes": "",
        "time": 1729183800000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=591",
        "notes": "",
        "time": 1729194600000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=636",
        "notes": "",
        "time": 1729281600000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=651",
        "notes": "",
        "time": 1729366200000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=673",
        "notes": "",
        "time": 1729452000000,
      },
    ],
    "title": "Joker Folie a Deux",
    "url": "https://www.genesiscinema.co.uk/event/87723",
  },
  {
    "overview": {
      "actors": [
        "Adam Driver",
        "Giancarlo Esposito",
        "Nathalie Emmanuel",
      ],
      "age-restriction": "15",
      "categories": [
        "Drama",
        "Sci-Fi",
      ],
      "directors": [
        "Francis Ford Coppola",
      ],
      "duration": 8280000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=565",
        "notes": "",
        "time": 1729001700000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=564",
        "notes": "",
        "time": 1729021800000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=576",
        "notes": "",
        "time": 1729088100000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=580",
        "notes": "",
        "time": 1729108200000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=635",
        "notes": "",
        "time": 1729270800000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=650",
        "notes": "",
        "time": 1729356600000,
      },
    ],
    "title": "Megalopolis",
    "url": "https://www.genesiscinema.co.uk/event/89986",
  },
  {
    "overview": {
      "actors": [],
      "categories": [
        "Comedy",
      ],
      "directors": [],
      "duration": 9000000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=479",
        "notes": "",
        "time": 1729017000000,
      },
    ],
    "title": "Slept On: An Alternative Comedy Night Hosted by Finlay & Joe",
    "url": "https://www.genesiscinema.co.uk/event/92484",
  },
  {
    "overview": {
      "actors": [
        "David Howard Thornton",
        "Lauren LaVera",
        "Jason Patric",
      ],
      "age-restriction": "18",
      "categories": [
        "Horror",
      ],
      "directors": [
        "Damien Leone",
      ],
      "duration": 7500000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=557",
        "notes": "",
        "time": 1729011900000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=558",
        "notes": "",
        "time": 1729022100000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=572",
        "notes": "",
        "time": 1729098000000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=573",
        "notes": "",
        "time": 1729108500000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=589",
        "notes": "",
        "time": 1729171800000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=588",
        "notes": "",
        "time": 1729195200000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=634",
        "notes": "",
        "time": 1729281900000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=649",
        "notes": "",
        "time": 1729368000000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=669",
        "notes": "",
        "time": 1729453500000,
      },
    ],
    "title": "Terrifier 3",
    "url": "https://www.genesiscinema.co.uk/event/91452",
  },
  {
    "overview": {
      "actors": [
        "Saoirse Ronan",
        "Saskia Reeves",
        "Stephen Dillane",
      ],
      "age-restriction": "15",
      "categories": [
        "Drama",
      ],
      "directors": [
        "Nora Fingscheidt",
      ],
      "duration": 7080000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=562",
        "notes": "",
        "time": 1729000800000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=566",
        "notes": "",
        "time": 1729012500000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=577",
        "notes": "",
        "time": 1729087200000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=578",
        "notes": "",
        "time": 1729098900000,
      },
    ],
    "title": "The Outrun",
    "url": "https://www.genesiscinema.co.uk/event/90363",
  },
  {
    "overview": {
      "actors": [
        "Margaret Qualley",
        "Dennis Quaid",
        "Demi Moore",
      ],
      "age-restriction": "18",
      "categories": [
        "Drama",
        "Horror",
      ],
      "directors": [
        "Coralie Fargeat",
      ],
      "duration": 8400000,
      "trailer": "https://www.youtube.com/watch?v=LNlrGhBpYjc",
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=563",
        "notes": "",
        "time": 1729011000000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=561",
        "notes": "",
        "time": 1729021500000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=579",
        "notes": "",
        "time": 1729097400000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=575",
        "notes": "",
        "time": 1729107900000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=593",
        "notes": "",
        "time": 1729194000000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=731",
        "notes": "",
        "time": 1729260000000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=656",
        "notes": "",
        "time": 1729270800000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=633",
        "notes": "",
        "time": 1729280700000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=655",
        "notes": "",
        "time": 1729354800000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=648",
        "notes": "",
        "time": 1729367100000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=675",
        "notes": "",
        "time": 1729431000000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=674",
        "notes": "",
        "time": 1729442400000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=666",
        "notes": "",
        "time": 1729452000000,
      },
    ],
    "title": "The Substance",
    "url": "https://www.genesiscinema.co.uk/event/89991",
  },
  {
    "overview": {
      "actors": [
        "Chris Hemsworth",
        "Brian Tyree Henry",
        "Scarlett Johansson",
      ],
      "age-restriction": "PG",
      "categories": [
        "Action",
        "Animation",
        "Science Fiction",
      ],
      "directors": [
        "Josh Cooley",
      ],
      "duration": 6240000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=559",
        "notes": "subtitles",
        "time": 1729003800000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=560",
        "notes": "",
        "time": 1729012500000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=733",
        "notes": "",
        "time": 1729023000000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=574",
        "notes": "",
        "time": 1729090200000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=504",
        "notes": "",
        "time": 1729098900000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=735",
        "notes": "",
        "time": 1729113900000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=586",
        "notes": "",
        "time": 1729177200000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=587",
        "notes": "subtitles",
        "time": 1729186200000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=734",
        "notes": "",
        "time": 1729194600000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=657",
        "notes": "",
        "time": 1729262100000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=736",
        "notes": "",
        "time": 1729282200000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=653",
        "notes": "",
        "time": 1729336800000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=654",
        "notes": "",
        "time": 1729348200000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=737",
        "notes": "",
        "time": 1729362300000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=663",
        "notes": "",
        "time": 1729422600000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=660",
        "notes": "",
        "time": 1729429200000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=738",
        "notes": "",
        "time": 1729448400000,
      },
    ],
    "title": "Transformers One",
    "url": "https://www.genesiscinema.co.uk/event/89379",
  },
  {
    "overview": {
      "actors": [
        "Paolo Malco",
        "Catriona Maccoll",
        "Giovanni Frezza",
        "Giovanni De Nava",
        "Dagmar Lassander",
        "Ania Pieroni",
        "Carlo De Mejo",
        "Silvia Collatina",
        "Daniela Doria",
        "Ranieri Ferrara",
        "Lucio Fulci",
        "Elmer Johnsson",
        "John Olson",
        "Teresa Rossi Passante",
        "Gianpaolo Saccarola",
      ],
      "age-restriction": "18",
      "categories": [
        "Horror",
      ],
      "directors": [
        "Lucio Fulci",
      ],
      "duration": 8700000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=480",
        "notes": "",
        "time": 1729107900000,
      },
    ],
    "title": "Bar Trash: The House By The Cemetery: 4K Restoration (1981)",
    "url": "https://www.genesiscinema.co.uk/event/92485",
  },
  {
    "overview": {
      "actors": [],
      "categories": [],
      "directors": [
        "Andrew Leavold",
      ],
      "duration": 4200000,
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=482",
        "notes": "",
        "time": 1729195200000,
      },
    ],
    "title": "FILM SAFARI GHANA: THE CHRONICLES OF NINJA + Q&A Presented by LAFS",
    "url": "https://www.genesiscinema.co.uk/event/92487",
  },
  {
    "overview": {
      "actors": [
        "Donald Pleasence",
        "Jamie Lee Curtis",
        "P.J. Soles",
        "Kyle Richards",
        "Nick Castle",
      ],
      "age-restriction": "15",
      "categories": [
        "Horror",
      ],
      "directors": [
        "John Carpenter",
      ],
      "duration": 5460000,
      "trailer": "https://www.youtube.com/watch?v=T5ke9IPTIJQ",
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=481",
        "notes": "",
        "time": 1729186800000,
      },
    ],
    "title": "Halloween Presented by the Cult Classic Collective",
    "url": "https://www.genesiscinema.co.uk/event/92486",
  },
  {
    "overview": {
      "actors": [
        "Rosemarie DeWitt",
        "Naomi Scott",
        "Kyle Gallner",
      ],
      "age-restriction": "18",
      "categories": [
        "Horror",
        "Mystery",
        "Thriller",
      ],
      "directors": [
        "Parker Finn",
      ],
      "duration": 6900000,
      "trailer": "https://www.youtube.com/watch?v=GaDd9PQBkdw",
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=583",
        "notes": "",
        "time": 1729184700000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=584",
        "notes": "",
        "time": 1729195200000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=639",
        "notes": "",
        "time": 1729260000000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=638",
        "notes": "",
        "time": 1729270500000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=637",
        "notes": "",
        "time": 1729281600000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=644",
        "notes": "",
        "time": 1729345800000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=643",
        "notes": "",
        "time": 1729356300000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=642",
        "notes": "",
        "time": 1729368000000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=671",
        "notes": "",
        "time": 1729431900000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=665",
        "notes": "",
        "time": 1729441500000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=659",
        "notes": "",
        "time": 1729452600000,
      },
    ],
    "title": "Smile 2",
    "url": "https://www.genesiscinema.co.uk/event/90365",
  },
  {
    "overview": {
      "actors": [],
      "categories": [],
      "directors": [],
      "duration": 7500000,
    },
    "performances": [
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=483",
        "notes": "",
        "time": 1729276200000,
      },
    ],
    "title": "Some Like it Swing",
    "url": "https://www.genesiscinema.co.uk/event/92488",
  },
  {
    "overview": {
      "actors": [
        "Sebastian Stan",
        "Jeremy Strong",
        "Maria Bakalova",
      ],
      "age-restriction": "15",
      "categories": [
        "Biography",
        "Drama",
        "History",
      ],
      "directors": [
        "Ali Abbasi",
      ],
      "duration": 7200000,
      "trailer": "https://www.youtube.com/watch?v=bvPRxy9kmSg",
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=628",
        "notes": "",
        "time": 1729261800000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=629",
        "notes": "",
        "time": 1729271700000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=630",
        "notes": "",
        "time": 1729281000000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=658",
        "notes": "",
        "time": 1729337400000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=640",
        "notes": "",
        "time": 1729344600000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=641",
        "notes": "",
        "time": 1729357500000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=645",
        "notes": "",
        "time": 1729367700000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=668",
        "notes": "",
        "time": 1729422000000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=672",
        "notes": "",
        "time": 1729431600000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=661",
        "notes": "",
        "time": 1729443000000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=664",
        "notes": "",
        "time": 1729453200000,
      },
    ],
    "title": "The Apprentice",
    "url": "https://www.genesiscinema.co.uk/event/90386",
  },
  {
    "overview": {
      "actors": [],
      "age-restriction": "U",
      "categories": [
        "Animation",
        "Adventure",
        "Family",
      ],
      "directors": [
        "Chris Sanders",
      ],
      "duration": 6060000,
      "trailer": "https://www.youtube.com/watch?v=njPNg0A9VpY",
      "year": "2024",
    },
    "performances": [
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=631",
        "notes": "",
        "time": 1729263000000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=632",
        "notes": "",
        "time": 1729271700000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=646",
        "notes": "",
        "time": 1729339200000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=652",
        "notes": "",
        "time": 1729347300000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=647",
        "notes": "",
        "time": 1729358700000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=662",
        "notes": "",
        "time": 1729425000000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=667",
        "notes": "",
        "time": 1729434000000,
      },
      {
        "bookingUrl": "https://genesis.admit-one.co.uk/seats/?perfCode=732",
        "notes": "",
        "time": 1729443000000,
      },
    ],
    "title": "The Wild Robot",
    "url": "https://www.genesiscinema.co.uk/event/85814",
  },
]
`;
